spec/tests/require.fmf

```diff
@@ -62,23 +71,42 @@ example:
   - |
     # Library from a custom remote git repository
     require:
-      - url: https://github.com/beakerlib/openssl
+      - type: library
+        url: https://github.com/beakerlib/openssl
         name: /certgen
 
   - |
     # Library from the local filesystem
     require:
-      - path: /tmp/library/openssl
+      - type: library
+        path: /tmp/library/openssl
```

Comment:
The usage of absolute paths looks weird to me. Is this trully an absolute path of a filesystem or a path within a cloned repository?

Reply:
This part is not relevant to the patch, it's for local libraries so yeah path is used for absolute path in the filesystem, I'm not changing anything in this part
the pattern used for file require is relative to tmt root as mentioned in the documentation for this (couple lines up)